scipy/stats/tests/test_distributions.py

```diff
@@ -3083,6 +3083,34 @@ def test_logcdf(self):
         assert_allclose(stats.lognorm.logsf(x2-mu, s=sigma),
                         stats.norm.logsf(np.log(x2-mu)/sigma))
 
+    @pytest.fixture(scope='function')
+    def rng(self):
+        return np.random.default_rng(1234)
+
+    @pytest.mark.parametrize("rvs_shape", [.1, 2])
+    @pytest.mark.parametrize("rvs_loc", [-2, 0, 2])
+    @pytest.mark.parametrize("rvs_scale", [.2, 1, 5])
+    @pytest.mark.parametrize('fix_shape, fix_loc, fix_scale',
+                             [e for e in product((False, True), repeat=3)
+                              if False in e])
+    @np.errstate(invalid="ignore")
+    def test_fit_MLE_comp_optimzer(self, rvs_shape, rvs_loc, rvs_scale,
```

Comment:
Unless this would make the line too long, let's fix the typo.
```suggestion
    def test_fit_MLE_comp_optimizer(self, rvs_shape, rvs_loc, rvs_scale,
```

Reply:
Guilty of copy-pasting. I fixed it for the three other classes using it.